portfolio/src/main/webapp/script.js

```diff
@@ -58,17 +61,22 @@ function getFeedback() {
 }
 
 /** Creates an <li> element containing text. */
```

Comment:
Nit: now it creates a \<li> element containing _a comment_.

Reply:
Just changed